Realm/RLMUtil.mm

```diff
@@ -360,6 +360,15 @@ BOOL RLMIsRunningInPlayground() {
     return [[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."];
 }
 
+void RLMWorkaroundRadar31252694() {
+    SEL isNSArray = NSSelectorFromString([@[@"is", @"NSArray", @"_", @"_"] componentsJoinedByString:@""]);
+    IMP yes = imp_implementationWithBlock(^(__unused id array) { return YES; });
+    Method isInvalidated = class_getInstanceMethod(RLMArray.class, @selector(isInvalidated));
+    const char *typeEncoding = method_getTypeEncoding(isInvalidated);
+    class_addMethod(RLMArray.class, isNSArray, yes, typeEncoding);
+    class_addMethod(RLMResults.class, isNSArray, yes, typeEncoding);
```

Comment:
I can easily add these 3 but I don’t know how to write the corresponding Swift unit tests.